src/css/header.css

```diff
@@ -1,23 +1,132 @@
-.header {
-  padding: 20px 0;
-  background-color: blue;
-  color: yellowgreen;
+/** ========= HEADER ========= */
+
+* {
+  box-sizing: border-box;
 }
 
-.nav-list {
+.header-container {
+  position: absolute;
+  width: 335px;
+  top: 20px;
+  left: 50%;
+  transform: translate(-50%);
   display: flex;
   align-items: center;
-  justify-content: flex-end;
-  gap: 40px;
+  justify-content: space-between;
+  border: 1px solid #fbfbfb;
+  border-radius: 8px;
+  background-color: #fbfbfb;
+  padding: 12px 20px;
 }
 
-.nav-link {
+.header-logo {
+  font-family: 'Inter', sans-serif;
   font-weight: 500;
-  letter-spacing: 0.02em;
-  line-height: 1.5;
+  font-size: 14px;
+  letter-spacing: -0.03em;
+  color: #1b414b;
+  display: flex;
+  align-items: center;
+  width: 52px;
+  height: 18px;
+}
+
+.burger-btn {
+  padding: 0;
+  border: none;
+  background-color: transparent;
+}
+
+.burger-icon {
+  fill: transparent;
+  stroke: #1b414b;
+}
+
+.header-nav-list,
+.header-soc {
+  display: none;
+}
+
+@media screen and (min-width: 768px) {
+  .header-container {
+    width: 704px;
+    padding: 12px 28px;
+  }
+
+  .header-logo {
+    width: 74px;
+    height: 24px;
+    font-size: 20px;
+    margin-right: 113px;
```

Comment:
now it can be, but think a little bit to use another way for this. at least one rule for positioning center that ul, try all what you know (maybe space-between, but in this case you must have 1 parent form all elements) ;)